src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -122,6 +124,9 @@ public class LogManagerService extends PluginService {
     @Getter
     private int periodicUpdateIntervalSec;
     private Future<?> spaceManagementThread;
+    // public only for integ tests
```

Comment:
Remove this comment, also this should not be public. See the comment below, we should only expose the register listener